cfn-resources/scripts/update-sdk.sh

```diff
@@ -22,4 +22,9 @@ LATEST_SDK_RELEASE=$(echo "${LATEST_SDK_TAG}" | cut -d '.' -f 1)
 echo "==> Updating SDK to latest major version ${LATEST_SDK_TAG}"
 gomajor get "go.mongodb.org/atlas-sdk/${LATEST_SDK_RELEASE}@${LATEST_SDK_TAG}"
 go mod tidy
+
+LATEST_SDK_STRIPPED_MAYOR_VERSION="${LATEST_SDK_RELEASE%%.*}"
+echo "==> Adjusting version defined in mockery file to ${LATEST_SDK_STRIPPED_MAYOR_VERSION}"
+perl -i -pe "s|go.mongodb.org/atlas-sdk/v[0-9]{11}/admin|go.mongodb.org/atlas-sdk/${LATEST_SDK_STRIPPED_MAYOR_VERSION}/admin|g" .mockery.yaml
```

Comment:
How does mockery use this info?

Reply:
since we are mocking directly against the sdk, the major version is defined in the config file https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/.mockery.yaml#L9. When a major version is updated we also need to update this config file. Hope this clarifies the doubt.